### PR TITLE
Added payouts and pdf downloads

### PR DESCRIPTION
**Pull request by henobi**
This PR contains the following:
- Added payout example to readme-documentation
- Added pdf download functionality and added a section in the docs
- Added test cases
- Changed version to 2.1.0

I tested the code changes successfully.

### CODE REVIEW

File: src/Request/CreateRequest.php [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 namespace Barzahlen\Request;
 
+
```
Comment (martinseener) [inline src/Request/CreateRequest.php]: double newline can be removed

File: src/Request/InvalidateRequest.php [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 namespace Barzahlen\Request;
 
+
```
Comment (martinseener) [inline src/Request/InvalidateRequest.php]: same here (double newline)

**Comment by martinseener** [comment]
@henobi i have 2 double newlines, a rebase and the failing tests but nonetheless it's a great MR! Thanks alot for this! 👍 🥇

**Comment by henobi** [comment]
Everything is fixed now.

**Pull request merged**